sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialTests.cs

```diff
@@ -450,15 +450,8 @@ public void VerifyImdsRequestFailureWithValidJsonIdentityNotFoundErrorThrowsCUE(
             var pipeline = CredentialPipeline.GetInstance(options);
 
             ManagedIdentityCredential credential = InstrumentClient(new ManagedIdentityCredential("mock-client-id", pipeline, options));
```

Comment:
replace this line with the following, and I think it will resolve the test issue:

```suggestion
            ManagedIdentityCredential credential = InstrumentClient(new ManagedIdentityCredential(
                new ManagedIdentityClient(
                    new ManagedIdentityClientOptions() { Pipeline = pipeline, ManagedIdentityId = ManagedIdentityId.FromUserAssignedClientId("mock-client-id"), IsForceRefreshEnabled = true, Options = options })));pipeline, options));
```